tests/queries/0_stateless/02360_send_logs_level_colors.sh

```diff
@@ -13,7 +13,7 @@ function run()
     command=$1
     expect << EOF
 log_user 0
-set timeout 3
+set timeout 60
```

Comment:
BTW it is nice to have debug facility, this can be done for example like this:

```sh
diff --git a/tests/queries/0_stateless/02360_send_logs_level_colors.sh b/tests/queries/0_stateless/02360_send_logs_level_colors.sh
index 4e5ce057702..e826c807fd2 100755
--- a/tests/queries/0_stateless/02360_send_logs_level_colors.sh
+++ b/tests/queries/0_stateless/02360_send_logs_level_colors.sh
@@ -12,8 +12,10 @@ function run()
 {
     command=$1
     expect << EOF
+exp_internal -f $CLICKHOUSE_TMP/$(basename "${BASH_SOURCE[0]}").debuglog 0
+
 log_user 0
-set timeout 3
+set timeout 20
 match_max 100000
 
 spawn bash -c "$command"
```

And those files will be automatically dumped by `clickhouse-test`